packages/dialog/src/react/index.js

```diff
@@ -95,7 +96,8 @@ Dialog.propTypes = {
   modal: PropTypes.bool,
   onClose: PropTypes.func,
   tailPosition: PropTypes.oneOf(Object.values(Dialog.tailPositions)),
-  returnFocus: PropTypes.bool
+  returnFocus: PropTypes.bool,
+  contentStyles: PropTypes.object
```

Comment:
This is a new style override pattern.  Looks pretty specific to this particular case as well.  I'd like to avoid adding a new prop for every dom node that devs want to restyle.